### PR TITLE
OSSM-8004: Use our proxy

### PR DESCRIPTION
**Pull request by jwendell**

### CODE REVIEW

File: tests/integration/security/external_ca/main_test.go [hunk 1]
```diff
@@ -41,6 +41,8 @@ func TestMain(m *testing.M) {
 	// nolint: staticcheck
 	var certs []csrctrl.SignerRootCert
 	framework.NewSuite(m).
+		// https://issues.redhat.com/browse/OSSM-5978
```
Comment (dgn) [inline tests/integration/security/external_ca/main_test.go:44]: that issue is closed as Won't Do :/

File: tests/integration/security/sds_ingress/quic/ingress_test.go [hunk 2]
```diff
@@ -78,6 +78,8 @@ func TestTlsGatewaysWithQUIC(t *testing.T) {
 		NewTest(t).
 		RequiresSingleCluster().
 		Run(func(t framework.TestContext) {
+			// FIXME: https://issues.redhat.com/browse/OSSM-1282
```
Comment (dgn) [inline tests/integration/security/sds_ingress/quic/ingress_test.go:81]: also closed as Won't Do

File: tests/integration/security/sds_ingress/quic/ingress_test.go [hunk 3]
```diff
@@ -96,6 +98,8 @@ func TestMtlsGatewaysWithQUIC(t *testing.T) {
...
```
Comment (dgn) [inline tests/integration/security/sds_ingress/quic/ingress_test.go:101]: same here

**Review by dgn — COMMENTED** [review]
Great job Jonh! I think we only need two new tracking issues

**Comment by jwendell** [comment]
@dgn Yeah, I thought about it as well, my conclusion was to keep the old jiras in the comments so we remember why those tests are skipped every time we cherry-pick this change. If we create new ones, they will be closed in some triaging meeting, I guess :) What do you think?

**Comment by dgn** [comment]
> @dgn Yeah, I thought about it as well, my conclusion was to keep the old jiras in the comments so we remember why those tests are skipped every time we cherry-pick this change. If we create new ones, they will be closed in some triaging meeting, I guess :) What do you think?

Maybe you're right. Alright, let's merge as-is then!

**Review by dgn — APPROVED** [review]

**Pull request merged**